cherno/guider.py

```diff
@@ -416,72 +375,26 @@ async def fit(
         else:
             self.command.debug(offset=offset)
```

Comment:
I'm not sure if thise should be `if any(full_offset): ... self.command.warning(offset=full_offset)`

Reply:
I'm not sure either...that was a chunk I believe was cut/pasted from the earlier implementation...it's up to you!

Reply:
Let's leave it for now.